### ISSUE TITLE
CLIEngine results should include warning and error count

### ISSUE THREAD

**Issue by sindresorhus** [eslint/eslint#1732]
CLIEngine should return how many errors and warnings there were so every consumer don't have to reimplement [it](https://github.com/sindresorhus/grunt-eslint/issues/45#issue-55477201).

```js
var results = engine.executeOnFiles(this.filesSrc);
results.errorCount;
results.warningCount;
```

**Comment by gyandeeps** [comment]
working on it.
